File: 0035-search-insert-position/0035-search-insert-position.py
class Solution(object):
    def searchInsert(self, nums, target):
        if target in nums:
            return nums.index(target)
        else:
            nums.append(target)
            nums.sort()
            return nums.index(target)

result = Solution()
result.searchInsert([1,3,5,6],2)

File: 0035-search-insert-position/0035-search-insert-position.cpp
class Solution {
public:
    int searchInsert(vector<int>& nums, int target){
        bool find = false;
        for (int i : nums) {
            if (i == target) {
                find = true;
                break;
            }
        }
        if (find == true) {
            for (int i = 0;i < nums.size();i++) {
                if (nums[i] == target) {
                    return i;
                }
            }
        }
        else {
            nums.push_back(target);
            sort(nums.begin(), nums.end());
            for (int i = 0;i < nums.size();i++) {
                if (nums[i] == target) {
                    return i;
                }
            }
        }
        return 0;
    }
};